src/components/ui/field.tsx

```
@@ -0,0 +1,238 @@
+    },
+  }
+)
+
+function Field({
+  className,
+  orientation = "vertical",
+  ...props
+}: React.ComponentProps<"div"> & VariantProps<typeof fieldVariants>) {
+  return (
+    <div
+      role="group"
```

Biome (2.4.11)

[error] The elements with this role can be changed to the following elements:
<fieldset>

(lint/a11y/useSemanticElements)